source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -70,211 +92,267 @@ Mixing
       VERSIONURL=<URL where the version of the mix will be hosted>
       FORMAT=1
 
```

Comment:
This builder.conf is missing the [Server] and [Mixer] sections:

[Mixer]
LOCAL_BUNDLE_DIR=/home/clr/mix/local-bundles

[Server]
debuginfo_banned=true
debuginfo_lib=/usr/lib/debug/
debuginfo_src=/usr/src/debug/

Reply:
@bktan8 Thanks for taking a look at this.  Yes, this is one of the changes coming in the update for 4.0